k8s/dockerfiles/graphscope-dev.Dockerfile

```diff
@@ -7,17 +7,10 @@ FROM $REGISTRY/graphscope/graphscope-dev-base:$BUILDER_VERSION
 
 USER root
```

Comment:
in k8s/dockerfiles/graphscope-dev-base.Dockerfile line-36, 
you are switched user to graphscope. 
Is there any must reasons to switch to root here? -- and then switch to graphscope again at last.